ravenml/data/interfaces.py

```diff
@@ -23,6 +34,98 @@ class CreateInput(object):
     for getting additional information.
 
     """
+    def __init__(self, config:dict=None, plugin_name:str=None):
+
+        if config is None or plugin_name is None:
+            raise click.exceptions.UsageError(('You must provide the --config option '
+                'on `ravenml create` when using this plugin command.'))
+        
+        self.config = config
+        
+        ## Set up Local Cache
+        # TODO: maybe create the subdir here?
+        # currently the cache_name subdir is only created IF the plugin places files there
+        self.plugin_cache = RMLCache(f'data_{plugin_name})
```

Comment:
Note I have altered this to avoid a conflict with the `TrainInput` constructor, since both dataset plugins and training plugins are exposing their commands with the same name (i.e `tf-bbox`). Thus their caches would be the same. By prepending the name with `data_` here and then `train_` in `TrainInput` we avoid any such conflicts.